File: coin-sums.cpp
#include <iostream>
using namespace std;

int main() {
    int cents;
    cin >> cents;

    int quarters = cents / 25;
    cout << "Quarters: " << quarters << endl;
    cents %= 25;

    int dimes = cents / 10;
    cout << "Dimes: " << dimes << endl;
    cents %= 10;

    int nickles = cents / 5;
    cout << "Nickles: " << nickles << endl;
    cents %= 5;

    int pennies = cents;
    cout << "Pennies: " << pennies << endl;

    return 0;
}

File: coin-sums.py
def coin_sums(cents):
    coins = [25, 10, 5, 1]
    quarters = cents // 25
    remaining = cents % 25
    nickles = remaining // 10
    remaining = remaining % 10
    dimes = remaining // 5
    remaining = remaining % 5
    pennies = remaining
    return quarters, nickles, dimes, pennies


cents = int(input())
quarters, nickles, dimes, pennies = coin_sums(cents)
print(quarters)
print(nickles)
print(dimes)
print(pennies)